components/MainContentContainer.vue

```diff
@@ -4,9 +4,13 @@ import SearchResultDetails from './SearchResultDetails.vue';
 <template>
     <div class="flex flex-1 bg-secondary-bg/20 hover:shadow-inner hover:shadow-secondary-bg/90">
         <!-- <WelcomeSection /> -->
-        <Modal class="min-h-1/2 ml-8 mt-12">
+        <Modal v-show="useModalStore().$state.isOpen" class="min-h-1/2 ml-8 mt-12">
```

Comment:
see other PR comments first

Reply:
@ermish These will be updated when I rebase off #267